ra2ce/analysis/losses/losses_base.py

```diff
@@ -157,10 +159,10 @@ def _get_disrupted_criticality_analysis_results(
         ]
         # link_id from list to tuple
         if len(self.criticality_analysis_non_disrupted) > 0:
-            self.criticality_analysis_non_disrupted[self.link_id] = (
-                self.criticality_analysis_non_disrupted[self.link_id].apply(
-                    lambda x: tuple(x) if isinstance(x, list) else x
-                )
+            self.criticality_analysis_non_disrupted[
```

Comment:
Plz solve linting issues on defining properties outsite the init.

Reply:
Can you please elaborate on this. Do you mean that we have to define the properties in the ini? In this case it gets input from after the analysis is executed